frigate/__main__.py

```diff
@@ -12,6 +15,8 @@
 cli.show_server_banner = lambda *x: None
 
 if __name__ == "__main__":
+    from frigate.app import FrigateApp
```

Comment:
This has to be moved from a top level import down here due to the restrictions imposed by Google Coral where we can't upgrade the distribution or python version and the pre-packaged sqlite3 library is too old, so we have to patch it.

Reply:
what version of sqlite3 does this enable frigate to be using?

Reply:
Latest if we bump the binary dep to 0.5.3 (released 3 days ago). Right now looks like 0.5.2 is a build around September 2023: https://github.com/coleifer/pysqlite3

Reply:
Update: With the latest changes we are manually building the latest version of sqlite3 and creating a self-contained wheel since `pysqlite3-binary` only distributes x86-64 wheels, so the answer to this question is now "always the latest release at build time"